dracut.sh

```diff
@@ -1334,6 +1334,9 @@ if ! [[ $print_cmdline ]]; then
     rm -fr -- "${initdir:?}"/*
 fi
 
+# This kernel doesn't support initramfs, so don't generate one
+check_kernel_config CONFIG_BLK_DEV_INITRD || exit 0
```

Comment:
We should `echo` some info message here so the user knows why dracut is stopping so early.